--- origsrc/blokish/configure.in	2006-01-07 16:23:16.000000000 -0600
+++ src/blokish/configure.in	2008-06-30 22:35:12.562500000 -0500
@@ -27,13 +27,8 @@
 AC_CHECK_FUNCS([sqrt])
 
 # Check for wxWidgets
-vers=`wx-config --version | awk 'BEGIN { FS = "."; } { printf "% d", ($1 * 1000 + $2) * 1000 + $3;}'`
-AC_MSG_CHECKING(for wxWidgets 2.6.0 or later)
-if test "$vers" -lt 2006000; then
-  AC_MSG_ERROR(no)
-else
-  AC_MSG_RESULT(yes)
-fi
+AM_OPTIONS_WXCONFIG
+AM_PATH_WXCONFIG(2.6.0, [], [AC_MSG_ERROR([$PACKAGE requires wxWidgets 2.6.x])], [core,html,gl])
 
 AC_CONFIG_FILES([makefile])
 AC_OUTPUT
--- origsrc/blokish/makefile.am	2006-01-04 04:39:29.000000000 -0600
+++ src/blokish/makefile.am	2008-06-30 22:03:35.500000000 -0500
@@ -1,5 +1,5 @@
-INCLUDES = `wx-config --cxxflags`
-LIBS = `wx-config --libs --gl-libs`
+INCLUDES = $(WX_CFLAGS)
+LDADD = $(WX_LIBS)
 
 bin_PROGRAMS = blokish
 
